lambda/translprim.ml

```diff
@@ -1968,6 +1981,7 @@ let lambda_primitive_needs_event_after = function
   | Pdls_get
   | Pobj_magic _ | Punbox_float _ | Punbox_int _ | Punbox_vector _
   | Preinterpret_unboxed_int64_as_tagged_int63 | Ppeek _ | Ppoke _
+  | Pget_idx _ | Pset_idx _
```

Comment:
I think these are analagous to the unsafe array operations - they are compiled to C calls that have camlasserts in them, on bytecode - so we should return `true` for them.